examples/intro/inducing_path.py

```diff
@@ -0,0 +1,120 @@
+"""
+======================================================
+An introduction to Inducing Paths and how to find them
+======================================================
+
+A path p is called an ``inducing path`` relateve to <L,S>
+on an ancestral graph, where every non-endpoint vertex on p
+is either in L or a collider, and every collider on p is an ancestor
+of either X , Y or a member of S.
+
+
+In other words, it is a path between two nodes that cannot be
+d-seperated, making it active regardless of what variables
+we condition on.
+
+More details on inducing paths can be found at :footcite:Zhang2008.
+
+References
+----------
+.. footbibliography::
```

Comment:
This doesn't render properly. I think you might need to add this to the very bottom of the file.

https://output.circle-artifacts.com/output/job/9f6f31fd-869c-41c9-83ce-c66a5107c2d6/artifacts/0/dev/auto_examples/intro/inducing_path.html#sphx-glr-auto-examples-intro-inducing-path-py